resources/js/components/FormField.vue

```diff
@@ -124,6 +124,20 @@
 						return;
 					}
 
+                                        if (Array.isArray(dependency.value)) {
```

Comment:
I think instead of adding a whole other case to worry about, we should just have the normal dependsOn case singular value just be upgraded to support arrays as values. I think there's also some misses here, specifically on L127, you are using `dependency.value` but you don't check if it exists until L129 which would make me ask 1. Why check after you are assuming it's there? 2. If we can assume it's there, then why are we checking it at all?